api/v1/aerospikecluster_validating_webhook.go

```diff
@@ -1235,6 +1239,24 @@ func validateNamespaceConfig(
 	return nil
 }
 
+func validateMRTFields(nsConf map[string]interface{}) error {
+	mrtField := isMRTFieldSet(nsConf)
+	scEnabled := IsNSSCEnabled(nsConf)
+
+	if !scEnabled && mrtField {
+		return fmt.Errorf("MRT fields are not allowed in non-SC namespace %v", nsConf)
+	}
+
+	return nil
+}
+
+func isMRTFieldSet(nsConf map[string]interface{}) bool {
+	_, isMRTDurationSet := nsConf["mrt-duration"]
+	disableMRTWrites, ok := nsConf["disable-mrt-writes"]
+
+	return (ok && disableMRTWrites.(bool)) || isMRTDurationSet
```

Comment:
In AP cluster if user sets `disable-mrt-writes` to false, validation will pass but reconcile will fail, as set-config command will fail.